### ISSUE TITLE
Image field labels create duplicate screen reader announcements

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov#504]
### Impact

Images are described by ‘field labels’. These labels are visually hidden but not hidden from screen reader announcements. This is a repetitive experience as user of screen reader heard “Image, [alt text] image” each time. 

### WCAG Fail 

This item is a fail 

### Expected results

User moves to image with a screen reader and only hears “Image” announced one and – where applicable the image alternative text. One announcement of image being the default screen reader announcement of the element role. 

### Suggested fix

Ensure any field labels do not reach the website front-end. Or, apply aria-hidden=”true” for all field labels. 

### Steps to reproduce

Inspect image in developer tools and notice visually hidden text preceding it. OR with a screen reader running (VoiceOver on Safari), arrow to any image in the page, before reaching the image listen for duplicated announcement of “image”. 

![image](https://user-images.githubusercontent.com/19519457/219045429-1c999b56-5c52-478f-93b7-b49e804a284b.png)

**Comment by keelanfh** [comment]
Closed as duplicate of https://github.com/localgovdrupal/localgov_core/issues/165
